### PR TITLE
UPDATE #231: Show .alignleft and .alignright as block elements on sma…

### LINKED ISSUE #231: .alignleft and .alignright are not positioned correctly on tablets and up
The `.alignleft` and `.alignright` elements in the entry content should align to the left and right edges of `.alignwide` elements on tablet screen sizes and up, as in the mockup below:

pull-quote-alignleft-alignment

This can be accomplished by using negative margins starting at the `700px` media query, like so:

```
.alignleft { 
    margin-left: calc( ( 100vw - 58rem - 8rem ) / -2 ); 
}

.alignright { 
    margin-right: calc( ( 100vw - 58rem - 8rem ) / -2 ); 
}
```

The calcs above take the screen width (`100vw`), subtracts the width of the content container (`- 58rem`), subtracts the left and right gutter of the layout (`- 8rem`), and finally divides the remainder in half and inverts it (`/ -2`), giving us the width difference between the `.entry-content` container and the `.alignwide` elements as a negative value.

Since it's based on the screen width, the CSS needs to be updated when the layout has reached it's max width, at `1280px` (there's a media query with that `min-width` currently containing some `.alignwide` CSS).  The CSS needed there is:

```
.alignleft { 
    margin-left: -31rem;
}

.alignright { 
    margin-right: -31rem;
}
```

### PR DESCRIPTION
**Pull request by nielslange**
Fixes #231 

<table>
<tr>
<td>Before (414px * 736px):
<br><br>

</td>
<td>After (414px * 736px):
<br><br>

</td>
</tr>
</table>

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -2107,6 +2107,11 @@ body.template-cover .entry-header {
 	font-family: 'Inter', -apple-system, BlinkMacSystemFont, 'Helvetica Neue', Helvetica, sans-serif;
 }
 
+.wp-block-pullquote.alignleft,
```
Comment (andersnoren) [inline style.css]: Can you move this block of CSS to beneath the "Block: Pullquote" CSS subheading, at line 2291?

File: style.css [hunk 2]
```diff
@@ -3353,6 +3358,44 @@ ul.footer-social li {
 /*	17. Media Queries
 /* --------------------------------------------------------------------------------------------- */
 
+@media ( min-width: 480px ) {
+
+	/* Block: Pullquote -------------------------- */
```
Comment (andersnoren) [inline style.css]: Rename to just "Blocks", for consistency with other media queries.

File: style.css [hunk 3]
```diff
@@ -3353,6 +3358,44 @@ ul.footer-social li {
...
+
+	/* STYLE: DEFAULT */
```
Comment (andersnoren) [inline style.css]: This is a bit of a special case... "BLOCK: BASE ALIGNMENT WIDTH" could maybe work, as it's similar to the "BLOCK: BASE MARGINS" subheading.

File: style.css [hunk 4]
```diff
@@ -3353,6 +3358,44 @@ ul.footer-social li {
...
+	.wp-block-pullquote.alignright,
+	.wp-block-cover-image.alignleft, 
+	.wp-block-cover-image.alignright,
+	.wp-block-cover.alignleft, 
+	.wp-block-cover.alignright,
+	.wp-block-embed.alignleft, 
+	.wp-block-embed.alignright,
+	.wp-block-gallery.alignleft, 
+	.wp-block-gallery.alignright {
+			max-width: 26rem
```
Comment (andersnoren) [inline style.css]: Seems like an extra tab here.

**Comment by andersnoren** [comment]
Also, it looks like this one doesn't contain the negative left and right margins that were in #231. :)

**Comment by nielslange** [comment]
@andersnoren With https://github.com/WordPress/twentytwenty/pull/291/commits/e8505e62ef1e5f024ef544990498fdcdbf404c97 I've added the lost margins as mentioned by you in https://github.com/WordPress/twentytwenty/pull/291#issuecomment-531658150,

**Comment by andersnoren** [comment]
Thanks @nielslange, looks good :)

**Pull request merged**